eyelid-match-ops/src/iris/conf.rs

```diff
@@ -95,6 +95,18 @@ const_assert!(FullRes::ROTATION_COMPARISONS <= FullRes::COLUMNS);
 const_assert!(FullRes::MATCH_NUMERATOR <= FullRes::MATCH_DENOMINATOR);
 const_assert!(FullRes::MATCH_DENOMINATOR > 0);
 
+// TODO: pick up these numbers from the report
+impl IrisConf for MiddleRes {
+    const COLUMN_LEN: usize = 5;
+    const COLUMNS: usize = 80;
```

Comment:
I think the report says 1024 coefficients with 5 rows of 160 bits:
```suggestion
    const COLUMNS: usize = 160;
```